.github/workflows/android.yaml

```diff
@@ -50,3 +50,13 @@ jobs:
             - name: Run Build
               run: |
                   scripts/build/gn_build.sh
+                  rsync -a out/lib/*.jar src/android/CHIPTool/app/libs
+                  rsync -a out/lib/jni/* src/android/CHIPTool/app/src/main/jniLibs
+                  git submodule update --init --recursive third_party/ot-commissioner/repo
+                  ABI=arm64-v8a API=21 ./third_party/ot-commissioner/build-android-libs.sh
+                # sdkmanager doesn't work with JDK 11.
```

Comment:
Then we shouldn't install Java 11. Let's only install the JDK that works and get rid of this hack.